deploy/examples/common-second-cluster.yaml

```
@@ -7,145 +7,145 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: $NAMESPACE
+  name: ceph-rook-secondary # namespace:cluster
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-cluster-mgmt
-  namespace: $NAMESPACE
+  namespace: ceph-rook-secondary # namespace:cluster
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: rook-ceph-cluster-mgmt
 subjects:
   - kind: ServiceAccount
     name: rook-ceph-system
-    namespace: rook-ceph
+    namespace: ceph-rook-secondary # namespace:operator
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-cmd-reporter
-  namespace: $NAMESPACE
+  namespace: ceph-rook-secondary # namespace:cluster
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: rook-ceph-cmd-reporter
 subjects:
   - kind: ServiceAccount
     name: rook-ceph-cmd-reporter
-    namespace: $NAMESPACE
+    namespace: ceph-rook-secondary # namespace:cluster
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-cmd-reporter-psp
-  namespace: $NAMESPACE
+  namespace: ceph-rook-secondary # namespace:cluster
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: psp:rook
 subjects:
   - kind: ServiceAccount
     name: rook-ceph-cmd-reporter
-    namespace: $NAMESPACE
+    namespace: ceph-rook-secondary # namespace:cluster
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-mgr-system
-  namespace: $NAMESPACE
+  namespace: ceph-rook-secondary # namespace:operator
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: rook-ceph-mgr-system
 subjects:
   - kind: ServiceAccount
     name: rook-ceph-mgr
-    namespace: $NAMESPACE
+    namespace: ceph-rook-secondary # namespace:cluster
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: rook-ceph-mgr-psp
-  namespace: $NAMESPACE
+  name: rook-ceph-mgr-psp # namespace:cluster
```

This tag should be one line lower
```suggestion
  name: rook-ceph-mgr-psp
```